user.go

```diff
@@ -113,3 +116,20 @@ func (c *Client) RetrieveMeWithContext(ctx context.Context) (out *User, err erro
 	c.get(ctx, "users", &out)
 	return
 }
+
+// UpdateBrand updates the user brand.
+func (c *Client) UpdateBrand(params map[string]interface{}, userID string) (out *Brand, err error) {
+	newParams := map[string]interface{}{"brand": params}
+	updateBrandURL := fmt.Sprintf("users/%s/brand", userID)
+	c.put(nil, updateBrandURL, newParams, &out)
+	return
+}
+
+// UpdateBrand performs the same operation as UpdateBrand, but allows
```

Comment:
Go docs expect the first word of the doc to be the function name, so here: `UpdateBrand` should become `UpdateBrandWithContext`.

Reply:
Wow, Godocs are really underwhelming for a modern language: https://go.dev/blog/godoc